src/Compilers/VisualBasic/Test/Semantic/Semantics/SelectCaseTests.vb

```diff
@@ -410,6 +410,56 @@ End Module
         End Sub
 
         <Fact()>
+        Public Sub SelectCase_RelationalCaseClauseExpression_IOperation()
+            Dim source = <![CDATA[
+Class C
+    Function LessThan(i As Integer, j As Integer) As Boolean
+        Select Case i
+            Case Is < j'BIND:"Is < j"
+                Return True
+            Case Else
+                Return False
+        End Select
+    End Function
+End Class]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IRelationalCaseClause (Relational operator kind: BinaryOperatorKind.LessThan) (CaseKind.Relational) (OperationKind.RelationalCaseClause) (Syntax: 'Is < j')
```

Comment:
Is `OperationKind.RelationalCaseClause` correct?

Reply:
No, the test needs to be fixed - I wanted to wait for everyone's review before fixing the test. Thanks.